apps/couch_httpd/src/couch_httpd.erl

```diff
@@ -121,16 +121,44 @@ start_link(Name, Options) ->
     % get the same value.
     couch_server:get_uuid(),
 
+    % add barrel log event handler
+    lager_handler_watcher:start(lager_event, barrel_log_event_handler, []),
+
+    WSLoop = fun([PayloadStr], Options, ReplyChannel) ->
```

Comment:
can you move it in its own function in barrel_websocket? Also barrel_websocket should be moved in barrel_httpd, barrel is only about core feature outside the HTTP api :)